package.json

```diff
@@ -32,6 +31,8 @@
     "@angular/platform-browser": "7.2.10",
     "@angular/platform-browser-dynamic": "7.2.10",
     "@angular/router": "7.2.10",
+    "@infragistics/igniteui-angular-charts": "^18.2.0-dev.1628",
+    "@infragistics/igniteui-angular-core": "^18.2.0-dev.1628",
```

Comment:
I'm not sure if we can use dev packages in public repo which will not be able to access them